.github/workflows/build-artifact.yaml

```diff
@@ -104,20 +115,40 @@ jobs:
           image: ${{ env.TT_METAL_DOCKER_IMAGE_TAG }}
           options: |
             --rm
+            --tmpfs /tmp
             -u ${{ env.RUNNER_UID }}:${{ env.RUNNER_GID }}
+            --group-add 1457
             -v ${{ github.workspace }}:${{ github.workspace }}
             -v /etc/passwd:/etc/passwd:ro
             -v /etc/shadow:/etc/shadow:ro
             -v /etc/bashrc:/etc/bashrc:ro
+            -v /home/ubuntu/.ccache-ci:/home/ubuntu/.ccache
+            -v /mnt/MLPerf/ccache:/mnt/MLPerf/ccache
             -e ARCH_NAME=${{ matrix.arch }}
             -w ${{ github.workspace }}
           run: |
-            build_command="./build_metal.sh --build-type ${{ inputs.build-type }} --build-tests --build-programming-examples"
+            # /tmp is a tmpfs; more efficient than persisted storage
+            mkdir -p /tmp/ccache
+            export CCACHE_TEMPDIR=/tmp/ccache
+
+            # Zero out the stats so we can see how we did this build
+            # NOTE: may be inaccurate if we have >1 build runner on the same machine, using the same local cache
+            ccache -z
```

Comment:
Do we care?

Reply:
Do we care that it's inaccurate?  Or to gather stats?

Inaccurate is unfortunate.  But not a top priority rn.  Gathering stats allows us to check the pulse.  Right now it's anecdotally to get a gut feel.  If we were to make it more reliable, then we could track the hit rate and see whether we fall off a cliff someday and know to go looking for a bad build setup (eg: introduction of ` __TIME__` macro in a header file, etc).

Reply:
Sounds good - we could actually upload this information as part of the CI/CD dashboard on superset as well